src/app.js

```diff
@@ -128,6 +128,15 @@ app.use((req, res, next) => {
   }
 });
 
+// Calculate and store the contact us URL so that every page template can access it.
+app.use((req, res, next) => {
+  const currentPageUrl = "https://signin.account.gov.uk" + req.originalUrl;
+  res.locals.contactUsUrl =
+    "https://home.account.gov.uk/contact-gov-uk-one-login?fromURL=" +
```

Comment:
Could we store this url at least in the config file and maybe pull this middleware out to its own function/file so it's easier to find?